CanalTP/Hermod
# Add version to status API

#### What I did

I add the tag (version) return for API status

#### How I did it

I set the env parameter version in parameters.yml (parameters.yml.dist) with the docker env file settings

#### How to verify it

Launch the docker build image (set the version number)
Get to API status
Check the version number
